domain/schema/model/sql/0011-storage.sql

```diff
@@ -261,6 +265,9 @@ CREATE TABLE storage_volume (
     hardware_id TEXT,
     wwn TEXT,
     persistent BOOLEAN,
+    obliterate_on_cleanup BOOLEAN,
```

Comment:
If we are going this way I would prefer that this column was not null and had a default value that represented our default intent.

Having 3 possible values a check constraint to understand is not ideal.

Reply:
Same feed back for filesystems.

Reply:
I did not want to introduce any (more) business logic into the constraint or the value.
So I did this with the intent that these decisions about the value it can have or what the "default" is done in business logic at the service layer of the storageprovisioning domain or the removal domain.